packages/cdk/src/code-build-project.ts

```diff
@@ -56,7 +56,10 @@ export class CodeBuildProject extends Construct {
       role: buildAsRole,
       source: gitHubSource,
       environment: {
-        buildImage: codebuild.LinuxBuildImage.STANDARD_6_0,
+        buildImage:
```

Comment:
don't bring the process.env.## all the way to here. add the node version as property only the interface `CodeBuildProjectProps` at line 9 above. And assign it in the code-pipeline-app.ts file form the process.env etc

Reply:
thanks, updated